### PR TITLE
fixes #6529 allow to define IP suggestion per subnet

### PR DESCRIPTION
**Pull request by ohadlevy**

### CODE REVIEW

**Comment by ohadlevy** [comment]
[test]

**Comment by dLobatog** [comment]
[test]

**Comment by dLobatog** [comment]
I've tested it and it looks good, my two concerns with this are:
- On the Host#new template, even if IPAM is disabled for the selected Subnet, I still see "suggest new" under the IP address field. I think it should be disabled when IPAM is disabled, the reason being when you click on it it just leaves the field empty without any feedback to the user, and it's a disabled functionality
- Next to the IP Address field on Host#new there is some inline help. I think it should mention you should configure it in the subnet in addition to have a DHCP proxy.

**Comment by domcleal** [comment]
New attributes should be added to API rabl responses too.

**Comment by ohadlevy** [comment]
@eLobato @domcleal updated

**Comment by dLobatog** [comment]
@ohadlevy I can't see the changes, "Suggest new" is shown when the selected Subnet has IPAM unchecked and "IP address auto-suggest" doesn't show how to enable IPAM, it only tells you to set up a DHCP proxy

File: app/assets/javascripts/host_edit.js [hunk 1]
```diff
@@ -220,8 +220,17 @@ function update_form(element, options) {
 }
 
 function subnet_selected(element){
+  var ipam_text =  $("#host_ip").parentsUntil('.clearfix').find(".help-block,.help-inline");
```
Comment (ohadlevy) [inline app/assets/javascripts/host_edit.js:223]: @eLobato this match catch both help block and help inline, not sure why it doesn't work for you? can you try it in the console?

**Comment by dLobatog** [comment]
It was a problem with my assets, it works well. Merged as https://github.com/theforeman/foreman/commit/160ffa2a85d5a2e6e9c0064c1b27e75c5108207f , thanks @ohadlevy !

**Pull request closed without merging**